shell/platform/android/external_view_embedder/external_view_embedder.cc

```diff
@@ -145,7 +145,8 @@ void AndroidExternalViewEmbedder::SubmitFrame(
       //
       // In this case, the rects are merged into a single one that is the union
       // of all the rects.
-      for (const SkRect& rect : intersection_rects) {
+      for (SkRect rect : intersection_rects) {
+        rect.intersect(current_view_rect);
```

Comment:
The fix should be similar to what iOS does: `joined_rect.intersect(platform_view_rect);`

Reply:
I agree with @cyanglaz here. A single intersect on the final joined rect should be enough per the comment I made in the iOS code.

Reply:
Yes, thanks for the response, you're right. 
Sorry, I didn't notice this on ios, but I just noticed it on Android and changed it synchronically. `joined_rect.intersect(platform_view_rect)`  it's a very good deal, I'll do it later.

Reply:
Done thanks~